pkg/solana/config/chain_reader.go

```diff
@@ -27,6 +31,9 @@ type ReadDefinition struct {
 	InputModifications  commoncodec.ModifiersConfig `json:"inputModifications,omitempty"`
 	OutputModifications commoncodec.ModifiersConfig `json:"outputModifications,omitempty"`
 	PDADefiniton        codec.PDATypeDef            `json:"pdaDefinition,omitempty"` // Only used for PDA account reads
+	IndexedFields       map[string]string           `json:"indexedFields,omitempty"`
+	// This will create a log poller filter for this event.
+	*PollingFilter `json:"pollingFilter,omitempty"`
```

Comment:
Ah, I see... this is trickier than I thought. We might need to change the config to have `IndexedFields` be a list of ordered pairs rather than a map 🤔

Either that, or we need enforce that they always get sorted using a deterministic sort before constructing the `logpoller.Filter` to pass to `RegisterFilter`, and in the event read binding before creating the `EventBySubKeyFilter`